frontend/server/src/DAO/Identities.php

```diff
@@ -55,7 +55,10 @@ public static function findByUsername(string $username): ?\OmegaUp\DAO\VO\Identi
         return new \OmegaUp\DAO\VO\Identities($rs);
     }
 
-    public static function findByUsernameOrName(string $usernameOrName): array {
+    /**
+     * @return array<int, \OmegaUp\DAO\VO\Identities>
```

Comment:
```suggestion
     * @return list<\OmegaUp\DAO\VO\Identities>
```

hay que cambiar L87 para que use `$result[] =` en vez de `array_push()`

Reply:
Ahh, eso explica por qué no podía utilizar una `list` normalita en la declaración.